M2/Macaulay2/m2/debugging.m2

```diff
@@ -139,7 +134,7 @@ show1(Sequence,Function) := show1(List,Function) := (types,pfun) -> (
 	       else install pfun v);
 --	  if hasAttribute(v,PrintNet) then v = getAttribute(v,PrintNet) else
 --	  if hasAttribute(v,PrintNames) then v = getAttribute(v,PrintNames) else
-	  if hasAttribute(v,ReverseDictionary) then v = getAttribute(v,ReverseDictionary);
+--	  if hasAttribute(v,ReverseDictionary) then v = getAttribute(v,ReverseDictionary);
```

Comment:
@DanGrayson What was the purpose of these three lines? Seems kind of silly now that all three are commented.

Reply:
yeah, none of them made sense...